financial-connections/src/main/java/com/stripe/android/financialconnections/analytics/FinancialConnectionsEvent.kt

```diff
@@ -1,416 +1,42 @@
 package com.stripe.android.financialconnections.analytics
 
-import com.stripe.android.financialconnections.domain.ConfirmVerification.OTPError
-import com.stripe.android.financialconnections.exception.FinancialConnectionsError
-import com.stripe.android.financialconnections.exception.WebAuthFlowFailedException
-import com.stripe.android.financialconnections.model.FinancialConnectionsSessionManifest.Pane
-import com.stripe.android.financialconnections.utils.filterNotNullValues
-
-/**
- * Event definitions for Financial Connections.
- */
-internal sealed class FinancialConnectionsEvent(
-    private val name: String,
-    val params: Map<String, String>? = null,
-    private val includePrefix: Boolean = true
+class FinancialConnectionsEvent internal constructor(
```

Comment:
No changes here -> 

- `FinancialConnectionsEvent` was renamed to `FinancialConnectionsInternalEvent`
- `FinancialConnectionsEvent` is now the new exposed event.